### PR TITLE
[694] LA elements and new default shadow

### PR DESCRIPTION
**Pull request by rlaco-qualityunit**
**Changes proposed in this Pull Request**
Boxes styles from LA web
Default box shadow was changed

**Testing instructions**
Check styles of new boxes elements from LA web (https://www.liveagent.com/collab-alternative/)
Check shadows on PAP theme pages

**Checklist**
- [x] My code follows the style guidelines of this project
- [x] My commits follow the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

Close QualityUnit/web-issues#694

### CODE REVIEW

File: assets/styles/common/_variables.scss [hunk 1]
```diff
@@ -134,7 +134,7 @@ $tag_orange-light: #fee3dc;
 $tag_red: #dc5d5d;
 $tag_red-light: #f6d6d6;
 
-$box-shadow: 0 0 16px 0 rgba(0, 0, 0, 0.08);
+$box-shadow: 0 0.938em 4.125em -0.938em rgba(9, 0, 44, 0.12);
```
Comment (jremen) [inline assets/styles/common/_variables.scss:137]: This is used across the web, beware of changes, especially EM units if some element has already defined font-size!

File: assets/styles/components/_Boxes.scss [hunk 2]
```diff
@@ -191,6 +220,128 @@
...
+					display: block;
+					margin: auto;
+					width: auto !important;
+					max-width: 100%;
+					// height: 100%;
+				}
+			}
+		}
+
+		.elementor-text-editor {
```
Comment (jremen) [inline assets/styles/components/_Boxes.scss:262]: elementor-widget-text-editor if possible, this is only existing class for streamlined elementor DOM we plan to deploy soon

File: assets/styles/components/_Boxes.scss [hunk 3]
```diff
@@ -191,6 +220,128 @@
...
+			}
+		}
+
+		.elementor-widget-heading,
+		.elementor-widget-text-editor {
+			position: static;
+			padding: 15px 0 0;
+
+			h3 {
+				font-size: 24px;
```
Comment (jremen) [inline assets/styles/components/_Boxes.scss:295]: use EM units and line-height recalculated accordingly

File: assets/styles/components/_Boxes.scss [hunk 4]
```diff
@@ -191,6 +220,128 @@
...
+				}
+			}
+		}
+	}
+
+	@media (max-width: $breakpoint-desktop) {
+
+		h2 {
+			font-size: $font-size-h2;
+			line-height: 42px !important;
```
Comment (jremen) [inline assets/styles/components/_Boxes.scss:316]: line-height: 1.3125 as $font-size-h2 == 32px so 42/32 = 1.3125

File: assets/styles/components/_Boxes.scss [hunk 5]
```diff
@@ -191,6 +220,128 @@
...
+		}
+
+		.elementor-container {
+			padding-left: 5px;
+			padding-right: 5px;
+		}
+
+		.elementor-col-50,
+		.elementor-col-33 {
+			padding: 15px;
```
Comment (jremen) [inline assets/styles/components/_Boxes.scss:326]: 1rem for 16px (1 pixel is pointless)

**Review by jremen — CHANGES_REQUESTED** [review]
Change as requested

File: assets/styles/common/_variables.scss (hunk 1 above)
Comment (rlaco-qualityunit) [reply assets/styles/common/_variables.scss]: Shadow should be changed on whole site (change was requested by @laci-qualityunit).
Unit was changed to px.

File: assets/styles/components/_Boxes.scss (hunk 2 above)
Comment (rlaco-qualityunit) [reply assets/styles/components/_Boxes.scss]: Changed

File: assets/styles/components/_Boxes.scss (hunk 4 above)
Comment (rlaco-qualityunit) [reply assets/styles/components/_Boxes.scss]: Changed

**Review by jremen — APPROVED** [review]

**Pull request merged**